spec/clipboard.spec.js

```diff
@@ -129,5 +129,16 @@ describe('Clipboard', () => {
 
       expect(parseContent(div)[0]).toEqual('&lt;b&gt;a&lt;/b&gt;')
     })
+
+    it('removes blacklisted HTML elements (e.g. <style>)', () => {
+      const div = document.createElement('div')
+      div.innerHTML = `<style type="text/css">
+        p.p1 {font: 30.0px Georgia; color: #000000; -webkit-text-stroke: #000000}
```

Comment:
I'm a fan of minimal tests. You could simplify this one a bit.

E.g
```js
div.innerHTML = `<style type="text/css">
    .foo { color: red; }
   </style>
   <p class="foo">
     bar
   </p>`

expect(parseContent(div)[0]).toEqual('bar')
```

Reply:
Fixed!